### PR TITLE
Update for training on GPU 3090

### PR DESCRIPTION
**Pull request by jinsongpan**
## what's the purpose of the changes
 Unable to train normally on GPU3090 using the latest commit.

## modified:
tip1: The default backends of torchaudio>=0.8.0 is changed to"sox_io"; so need set the audio backend as "sox" for normal training; 
ref: https://pytorch.org/audio/stable/backend.html#backend  for full details.

tips2: "torchaudio.sox_effects.SoxEffectsChain()" is invalid in torchaudio>=0.8.0

tip3: yaml.load() without Loader=... is deprecated in yaml>5.1,  as the default Loader is unsafe;
ref: https://msg.pyyaml.org/load for full details.

### CODE REVIEW

**Comment by robin1001** [comment]
The updated API may works for torchaudio>=0.80, but it should fail for lower version. Maybe we can just specify the torchaudio with lower version

**Comment by robin1001** [comment]
I specified the torchaudio version in https://github.com/mobvoi/wenet/pull/245, and your changes may be adopted in the future for a big version update.

**Comment by jinsongpan** [comment]
> I specified the torchaudio version in #245, and your changes may be adopted in the future for a big version update.

Sorry, i didn't notice that "torchaudio.sox_effects.apply_effects_file()" did not supperted on torchaudio==0.6.0;
but i don't think  #245 is a good idea, because it still don't support GPU 3090; I advice not fixing the version to torchaudio==0.6.0 and adding a few lines of code in the dataset.py: if torchaudio<=0.6.0, using "SoxEffectsChain()"; else using "apply_effects_file()".
other changes are useful in lower version.

**Comment by robin1001** [comment]
Do you mean torchaudio 0.60 is not supported on 3090? Or torchaudio 0.60 could not cowork with higher pytorch version1.8?

File: wenet/dataset/dataset.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Copyright (c) 2020 Mobvoi Inc. (authors: Binbin Zhang, Chao Yang)
+# 2021 jinsong pan
```
Comment (robin1001) [inline wenet/dataset/dataset.py:2]: Copyright (c) 2021 Jinsong Pan

**Comment by jinsongpan** [comment]
> Do you mean torchaudio 0.60 is not supported on 3090? Or torchaudio 0.60 could not cowork with higher pytorch version1.8?

yes. I have updated the code to suport all torchaudio versions until now.

**Pull request closed without merging**